web/script.js

```diff
@@ -236,6 +238,19 @@
       $("#test").append(tcases.map(e => makeButton("test", e[0], makeTooltip(e[1]))));
     }
     setButtonState();
+
+    $("table.result").delegate("td", "mouseover mouseleave", function(e) {
+        const t = $(this).closest("table.result");
+        if (e.type === "mouseover") {
```

Comment:
I believe you can simplify this by using `toggleClass`, can't you?

Reply:
Or wait, can't you even do this in pure CSS? `tr:hover td, colgroup:hover td`?

Reply:
`toggleClass` for some reason didn't work. I don't know if this would work with pure CSS (not my strength...)

Reply:
Let me try though.

Reply:
`toggleClass` doesn't work, because the elements inside the `td` also have `hover` callbacks and something is getting confused.

I couldn't get CSS to work at all.

Reply:
It looks like it's hard to highlight a column. There are some CSS hacks to make it work, but they're kind of ugly.